apps/vs-agent/src/utils/data.ts

```diff
@@ -0,0 +1,236 @@
+import { PUBLIC_API_BASE_URL } from '../config'
```

Comment:
This will not work, since now we are encouraging users to only define AGENT_PUBLIC_DID. I think we must find a different way to render these schemas. Probably turning this into a function, so the publicApiBaseUrl is passed from the controller (which has access to Agent instance).